Kalyani/HTML/Assginment3/Qust1.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<head>
+    <title>
+        <h1>List/h1>
+    </title>
+</head>
+<body>
+    <h2>Ambersoft</h2>
+
+    <dl>
```

Comment:
at first there are two ordered list item so use 
orderd list. not <dd> type of list 

for good understanding watch any nested list element video on youtube..

Reply:
ohk